backend/index.js

```diff
@@ -16,11 +16,17 @@ require("dotenv").config();
 const usersRoute = require("./app/routes/UserController");
 const alertsRoute = require("./app/routes/AlertController");
 const authenticationRoute = require("./app/routes/Authentication");
+const daeRoute = require("./app/routes/DAEController");
+const clinicRoute = require("./app/routes/ClinicController");
+const hospitalRoute = require("./app/routes/HospitalController");
 
 // Route middleware
 app.use("/api/v1/users", usersRoute);
 app.use("/api/v1/alerts", alertsRoute);
 app.use("/api/v1/tokens", authenticationRoute);
+app.use("/api/v1/dae", daeRoute);
```

Comment:
in teoria le risorse vanno al plurale quindi qui sarebbe daes, clinics e hospitals

Reply:
yes, correggo subito